<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixInversion</class>
 <widget class="QDialog" name="MatrixInversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="source_matrix_widget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>4</width>
     <height>4</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>4</width>
     <height>4</height>
    </size>
   </property>
  </widget>
  <widget class="QTableWidget" name="reversed_matrix_widget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>100</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_source">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Початкова матриця</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_reversed">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Обернена матриця</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>651</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Записати у файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>291</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Отримати інформацію про алгоритм</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_done">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Готово</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>statusBar</zorder>
  <zorder>source_matrix_widget</zorder>
  <zorder>reversed_matrix_widget</zorder>
  <zorder>label_source</zorder>
  <zorder>label_reversed</zorder>
  <zorder>pushButton_write_1</zorder>
  <zorder>pushButton_inf</zorder>
  <zorder>pushButton_done</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
